File: Beelina.LIB/GraphQL/Errors/ClientSubscriptionNotExistsError.cs
using Beelina.LIB.Interfaces;

namespace Beelina.LIB.GraphQL.Errors
{
    public class ClientSubscriptionNotExistsError
        : BaseError, IClientSubscriptionDetailsPayload
    {
        public ClientSubscriptionNotExistsError()
        {
        }

        public ClientSubscriptionNotExistsError(string message)
        {
            Message = message;
        }
    }
}

File: Beelina.APP/src/app/_models/errors/client-subscription-not-exists.error.ts
import { IBaseError } from 'src/app/_interfaces/errors/ibase.error';
import { IClientSubscriptionDetailsPayload } from 'src/app/_interfaces/payloads/iclient-subscription-details.payload';

export class ClientSubscriptionNotExistsError
  implements IClientSubscriptionDetailsPayload, IBaseError {
  public typename: string;
  public code: string;
  public message: string;
}
